src/databricks/labs/remorph/cli.py

```diff
@@ -82,5 +83,21 @@ def reconcile(w: WorkspaceClient, recon_conf: str, conn_profile: str, source: st
     recon(recon_conf, conn_profile, source, report)
 
 
+@remorph.command
+def generate_lineage(w: WorkspaceClient, source: str, input_sql: str, output_folder: str):
+    """Generates a lineage of source SQL files or folder"""
+    logger.info(f"user: {w.current_user.me()}")
+    if source.lower() not in {"snowflake", "tsql"}:
```

Comment:
nit: create a common function as these checks are starting to repeat.

Reply:
Instead of hardcoding values, we should leverage  the existing class: `from databricks.labs.remorph.reconcile.constants import SourceType`

SourceType.SNOWFLAKE.value,
SourceType.ORACLE.value,
SourceType.DATABRICKS.value,
SourceType.NETEZZA.value.

Reference can be found here: https://github.com/databrickslabs/remorph/pull/223/files#diff-0599e0372a638eec3be65f3967c600d20f581c819205cbdb5007d8a1801922f1

Reply:
This is set up for only Snowflake and not TSQL, so I have not specified TSQL for now.

Reply:
ok, please make sure to use the `SourceType` class instead of hardcoding the actual value ("snowflake")

Reply:
we need to use engine every where, from a common application constant, write now this is missing so this will be taken as technical debt.
I want remorph `transpile` and `reconcile` to share same engine config.